README.md

```diff
@@ -0,0 +1,19 @@
+# chef
+Task for Epam Java Collection course
+
+In this project you can find salad making functionality. The program contains list of ingredients and two ready salads - Chef's and Russian salads.
+
+To launch the project using ide:
+Open the project and run main method in Main class.
+
+To launch the project without ide:
+1. Be sure you have downloaded JDK
+2. Download the project
+3. Open console and go to src project folder
+4. Run the command javac Main.java
```

Comment:
you can use readme.md syntax to highlight snippets of code or commands
`like this one`
https://www.markdownguide.org/basic-syntax/#code-blocks

Reply:
highlighted commands